sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentClassifierAdministrationLiveTests.cs

```diff
@@ -143,7 +143,7 @@ public async Task GetDocumentClassifier(bool useTokenCredential)
         {
             var client = CreateDocumentModelAdministrationClient(useTokenCredential);
             var classifierId = Recording.GenerateId();
-            var description = "This classifier was generated by a .NET test.";
+            var description = $"This classifier was generated by a .NET test: {nameof(GetDocumentClassifier)}";
```

Comment:
Adding the test name to some descriptions just to know where to look in case of model leaks (if we ever forget to delete a document model after test cleanup).